tf2_ros/doc/conf.py

```diff
@@ -36,9 +46,14 @@
 # The master toctree document.
 master_doc = 'index'
 
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path .
+exclude_patterns = []
+
 # General information about the project.
-project = u'tf'
-copyright = u'2009, Willow Garage, Inc.'
+project = u'tf2_ros'
+copyright = u'2025, Open Source Robotics Foundation'  # noqa(A001)
```

Comment:
Changing the project name makes sense, but a bunch of files still have the Willow Garage copyright notice, so I would leave that as-is.